charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -367,15 +373,19 @@ sshConfig:
     fsGroup: 0
     runAsUser: 0
 
-  env: []
+  env:
+    # This env variable instruct docker to start only ssh-proxy-bastion processes
+    - name: "BASTION"
+      value: "ssh-proxy"
+
   initContainer:
     image:
       repository: busybox
       tag: 1.30.1
       pullPolicy: IfNotPresent
 
   image:
-    repository: akeyless/ssh-proxy
+    repository: akeyless/zero-trust-portal
```

Comment:
Is this the correct repo?
Shouldn't the repo for `zero trust bastion` be the same?

Reply:
sshProxy and web bastion should have the same repo - `akeyless/zero-trust-bastion`, the portal stays the same (udpdated the code)